### PR TITLE
Update Japanese translation (Line 61-110)

### PR DESCRIPTION
**Pull request by t-inu**
/hold for Japanese developer's review.
/assign @atoato88 @yosshy
/cc @YuikoTakada @s-kawamura-w664

### CODE REVIEW

File: translations/ja.json [hunk 1]
```diff
@@ -58,55 +58,55 @@
...
+	"Alternatively you could install one of these drivers:": "代わりに、これらのドライバーのいずれかをインストールすることもできます:",
+	"Amount of RAM to allocate to Kubernetes (format: \u003cnumber\u003e[\u003cunit\u003e], where unit = b, k, m or g).": "Kubernetes に割り当てられた RAM 容量 (形式: \u003cnumber\u003e[\u003cunit\u003e]、unit = b、k、m、g)。",
+	"Amount of time to wait for a service in seconds": "サービスを待機する時間 (秒)",
+	"Amount of time to wait for service in seconds": "サービスを待機する時間 (秒)",
+	"Another hypervisor, such as VirtualBox, is conflicting with KVM. Please stop the other hypervisor, or use --driver to switch to it.": "VirtualBox などの別のハイパーバイザーが、KVM と競合しています。他のハイパーバイザーを停止するか、--driver を使用して切り替えてください。",
+	"Another minikube instance is downloading dependencies... ": "別の minikube のインスタンスが、依存関係をダウンロードしています... ",
+	"Another program is using a file required by minikube. If you are using Hyper-V, try stopping the minikube VM from within the Hyper-V manager": "別のプログラムが、minikube に必要なファイルを使用しています。Hyper-V を使用している場合は、Hyper-V マネージャー内から minikube を停止してみてください",
```
Comment (atoato88) [inline translations/ja.json:68]:
```suggestion
	"Another program is using a file required by minikube. If you are using Hyper-V, try stopping the minikube VM from within the Hyper-V manager": "別のプログラムが、minikube に必要なファイルを使用しています。Hyper-V を使用している場合は、Hyper-V マネージャー内から minikube VM を停止してみてください",
```

**Comment by atoato88** [comment]
Almost LGTM, I left just one comment.

File: translations/ja.json (hunk 1 above)
Comment (t-inu) [reply translations/ja.json]: Thanks! I fixed it.

**Comment by atoato88** [comment]
/lgtm
Thank you to update PR.

**Comment by t-inu** [comment]
@atoato88 Thank you for your review.
@sharifelgamal Would you please merge this PR?
/unhold
/assign @sharifelgamal

**Review by sharifelgamal — APPROVED** [review]

**Comment by t-inu** [comment]
@sharifelgamal Thanks for merging it!

**Pull request merged**